packages/react-components/src/components/styles/Dialpad.styles.ts

```diff
@@ -95,7 +95,11 @@ export const iconButtonStyles = (theme: Theme): IButtonStyles => {
       color: `${theme.palette.black}`
     },
     icon: {
-      height: 'auto'
+      height: 'auto',
+      // Needed to keep the icon vertically centered.
+      '> span': {
+        display: 'flex'
```

Comment:
I'm surprised this is working as the span inside the icon button already has a display flex type
![image](https://user-images.githubusercontent.com/97130533/193879102-d9f08dec-eaeb-424f-bc92-28b3ec384068.png)

Reply:
I did find that that changing the '> i': { fontSize: '20px' } resolves this issue.

Reply:
I did find doing the following fixed the issue:
![image](https://user-images.githubusercontent.com/97130533/193879993-1400e346-9e21-40f4-8977-b33316a919c9.png)

Reply:
:o
Strange. That explains why the snapshots were not generated

Reply:
nit: instead of using px, use 1.25rem.